### PR TITLE
Add Indic xnli pair classification

### PR DESCRIPTION
**Pull request by SaitejaUtpala**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

- [ ] I have tested that the dataset runs with the `mteb` package.
- [ ] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [ ] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [ ] `intfloat/multilingual-e5-small`
- [ ] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [ ] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [ ] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [ ] Run tests locally to make sure nothing is broken using `make test`. 
- [ ] Run the formatter to format the code using `make lint`. 
- [ ] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

**Comment by SaitejaUtpala** [comment]
waiting until #582

**Comment by imenelydiaker** [comment]
> waiting until #582

This is not really an issue, @loicmagne already answered, you need to adjust your dataset to meet the expected format by the PC task. Please checke existing examples fo PC tasks with `data_transform()` function that adjust datasets to the expected format.

We'll think about updating this format but for now it's not the priority for the moment. Note that this update would require to update all mteb/*PC datasets.

File: mteb/evaluation/evaluators/PairClassificationEvaluator.py [hunk 1]
```diff
@@ -45,8 +45,10 @@ def __init__(
 
         assert len(self.sentences1) == len(self.sentences2)
         assert len(self.sentences1) == len(self.labels)
-        for label in labels:
-            assert label == 0 or label == 1
+
+        # removing this indic xnli is 3-way (entailment, neutral, )
+        # for label in labels:
+        #     assert label == 0 or label == 1
```
Comment (imenelydiaker) [inline mteb/evaluation/evaluators/PairClassificationEvaluator.py:51]: Undo this change and filter out the neutral values just as it has ben done here: [XNLI](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/PairClassification/multilingual/XNLI.py)

**Comment by KennethEnevoldsen** [comment]
From the checklist it seems like this is still a work in progress. Will close it for now, but feel free to re-open it.
